mods/ts/rules/shared-support.yaml

```diff
@@ -9,6 +9,7 @@ NAPULS:
 		Queue: Defense
 		BuildPaletteOrder: 90
 		Prerequisites: radar
+		BuildLimit: 1
```

Comment:
I disagree original behaviour is much better. As i said more EMP Cannon doesn't mean more SW. As EMP Cannon has a range limit, others just provides ability to attack other places.